tasks/test/browser.js

```diff
@@ -37,11 +38,18 @@ module.exports = (gulp) => {
   gulp.task('test:browser', (done) => {
     const runSequence = require('run-sequence')
 
-    runSequence.use(gulp)(
-      'test:karma',
-      'test:karma:webworker',
-      utils.exitOnFail(done)
-    )
+    if (util.env.webworker === 'only') {
```

Comment:
I don't think we need to check for the flags here and in bin/test... I think its better if we do it in one place. IMHO, bin/test is probably better and the leaving tasks/test/browser the way it was before but with the new flag names.